ci/build_container/build_container.sh

```diff
@@ -19,6 +19,17 @@ apt-get update
 apt-get install -y bazel
 rm -rf /var/lib/apt/lists/*
 
+# Build a version of Bazel suitable for C++ code coverage collection. See
+# https://github.com/bazelbuild/bazel/issues/1118 for why we need this. This is the envoy-coverage
+# branch on the cloned repository.
+git clone https://github.com/htuch/bazel.git /tmp/bazel-coverage
```

Comment:
At this pint I point I think we rely on 2 branches you are maintaining to get bazel working (this and I think protobuf). Can we add getting whatever these fixes are merged to master to here: https://github.com/lyft/envoy/issues/415#issuecomment-292613447

These are both Google owned projects so presumably we can get the fixes merged. I would prefer to not be in the long term business of maintaining private branches. If this is needed for some reason, let's discuss OOB because we should fork them into Lyft org and do the fixes there IMO.